### PR TITLE
MSR - Update changelog for 2.0.0 stuff

### LINKED ISSUE #6701: Update Changelog for OSRR 2.0.0
PR https://github.com/randovania/randovania/pull/6632 had to update the patcher but there are many - totally to that pr unrelated - changes which needs to be added to the changelog.

### PR DESCRIPTION
**Pull request by dyceron**
Fixes #6701

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,6 +25,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Metroid: Samus Returns
 
 - Added: New door types: `Access Open` and `Lightning Armor`. Access Open adds new doors to most 3-Tile high open transitions, which can then be shuffled. Lightning Armor doors can be opened with a Lightning Melee.
+- Changed: The popup when collecting an item has been removed, and has been replaced with an asynchronous message that does not interrupt gameplay.
```
Comment (Miepee) [inline CHANGELOG.md:28]:
```suggestion
- Changed: The popup when collecting an item has been removed, and has been replaced with a message that does not interrupt gameplay.
```

async is tech jargon

File: CHANGELOG.md [hunk 2]
```diff
@@ -25,6 +25,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Changed: Area 1 - Inner Temple East Hall - The top crumble block no longer respawns to help prevent a possible softlock in Door Lock Rando.
+- Changed: When collecting a Reserve Tank, the HUD will now properly update instead of disabling the bottom screen.
+- Changed: Area 4 Crystal Mines - Tsumuri Tunnel - The collision of the right wall has been extended past the door, allowing for the upper door in Mines Entrance to be shuffled in Door Lock Rando.
```
Comment (Miepee) [inline CHANGELOG.md:31]:
```suggestion
- Changed: Area 4 Crystal Mines - Tsumuri Tunnel - The upper door in Mines Entrance can now be shuffled in Door Lock Rando.
```

Rest is unneeded info

File: CHANGELOG.md [hunk 3]
```diff
@@ -232,6 +238,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed: Starting at Area 3 Factory Exterior - Beam Burst Chamber & Tsumuri Station no longer spawns Samus out of bounds.
 - Fixed: The Laser Aim cosmetic options UI no longer exports the wrong colors and has been simplified.
 - Fixed: The item in Area 7 - Omega Arena South Access no longer disappears after defeating the Omega in Area 7 - Omega Arena South.
+- Fixed: Case where PBs would not be disabled when fighting Diggernaut.
```
Comment (Miepee) [inline CHANGELOG.md:241]:
```suggestion
- Fixed: Case where Power Bombs would not be disabled when fighting Diggernaut.
```

**Comment by dyceron** [comment]
Addressed

**Review by ThanatosGit — APPROVED** [review]

**Pull request merged**